test/setup.test.js

```diff
@@ -3,10 +3,25 @@
 // for unit testing outside of docker
 const Rapptor = require('rapptor');
 
-module.exports.withRapptor = (options, callback) => {
+module.exports.server = undefined;
+module.exports.withRapptor = (options, dataSet, callback) => {
```

Comment:
take a look at this: https://github.com/firstandthird/inspired-survey/blob/master/test/setup.test.js